Sources/TuistSigning/GraphMappers/SigningMapper.swift

```diff
@@ -93,3 +93,14 @@ public class SigningMapper: ProjectMapping {
         return target
     }
 }
+
+extension Dictionary where Key == Fingerprint, Value == Certificate {
+    func first(for provisioningProfile: ProvisioningProfile) -> Certificate? {
+        for fingerprint in provisioningProfile.developerCertificateFingerprints {
```

Comment:
I'd rewrite this to a more concise version: `provisioningProfile.developerCertificateFingerprints.compactMap { self[fingerPrint] }.first`